move/walrus_site/sources/site.move

```diff
@@ -26,7 +28,17 @@ module walrus_site::site {
         blob_id: u256,
         // Contains the hash of the contents of the blob
         // to verify its integrity.
-        blob_hash: u256
+        blob_hash: u256,
+        // Defines the byte range of the resource contents
+        // in the case where multiple resources are stored
+        // in the same blob. This way, each resource will
+        // be parsed using its' byte range in the blob.
+        range: Option<Range>,
+    }
+
+    public struct Range has store, drop {
```

Comment:
Question: does this need `store`?
```suggestion
    public struct Range has drop {
```

Reply:
Yea the `store` ability is required for every type in order to be stored in objects with the key ability (Resource).

Reply:
https://move-book.com/storage/store-ability.html